docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+      - .:/var/www/firefly-iii
```

Comment:
What you say sounds like what I would expect to happen, but I just ran setup, and the vendor folder was present and the app was working.